lib/aws_recon/version.rb

```diff
@@ -1,3 +1,3 @@
 module AwsRecon
-  VERSION = "0.5.31"
+  VERSION = "0.5.32"
```

Comment:
this is not needed, the gem version will get bumped automatically when it gets pushed to rubygems.